stdlib/2and3/argparse.pyi

```diff
@@ -121,8 +121,16 @@ class ArgumentParser(_AttributeHolder, _ActionsContainer):
                      argument_default: Optional[_Text] = ...,
                      conflict_handler: _Text = ...,
                      add_help: bool = ...) -> None: ...
+
+    @overload
     def parse_args(self, args: Optional[Sequence[_Text]] = ...,
                    namespace: Optional[Namespace] = ...) -> Namespace: ...
```

Comment:
I think this overload shouldn't have the `namespace` argument. If `namespace` is given, the second overload suffices.